src/beccaccino.rs

```diff
@@ -32,15 +32,7 @@ impl Game for Beccaccino {
     fn get_name(&self) -> &str {
         "Beccaccino"
     }
-    fn init() -> Self {
-        Self {
-            players: vec![],
-            in_hand: vec![vec![], vec![], vec![], vec![]],
-            briscola: None,
-            table: vec![],
-            won_cards: vec![(vec![], false), (vec![], false)],
-            next_player: None
-        }
+    fn init(&mut self) {
```

Comment:
This function now is used to reinit the object after it has been closed (i.e. reshuffle the deck)
The initialization of the object has been implemented by implementing the `Default` trait